Justfile

```diff
@@ -14,28 +14,19 @@ default:
 
 set dotenv-load := true
 
-tailwind:
-  npx tailwindcss -i assets/css/input.css -o assets/css/styles.css --minify
-
-tailwind-watch:
```

Comment:
We no longer need to explicitly watch for file changes. `trunk serve` will watch for changes in files and re-serve.